third_party/pyth/p2w_autoattest.py

```diff
@@ -114,7 +114,7 @@
 min_rpc_interval_ms: 0 # RIP RPC
 max_batch_jobs: 1000 # Where we're going there's no oomkiller
 default_attestation_conditions:
-  min_interval_secs: 60
+  min_interval_secs: 10
```

Comment:
10 seconds is still slow compared to the rest of groups but will help to make e2e tests run faster.